php/laravel/config/servel.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'servers' => [
+        'name' => 'http://0.0.0.0:80',
```

Comment:
should use `3000`

Reply:
Sure, the configuration I posted has a bug. Please change the content of `config/servel.php` to the following.

```php
<?php

return [
    'servers' => [
        [
            'name' => 'http://0.0.0.0:3000',
            'count' => 16 // It depends on the hardware, change it if necessary.
        ]
    ]
];
```

This will solve the execution problem, thanks.